runbot_render_logs/models/runbot_build.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+# © 2015 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp.addons.runbot.runbot import run
+
+from openerp import models
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotBuild(models.Model):
+    _inherit = 'runbot.build'
+
+    def find_port(self, cr, uid):
+        # currently used port
```

Comment:
Please, add a docstring of why we need a full overwrite of this method.
And add # comments where the code change
And # ## comment with section of code is from original method